docs/markdown/Reference-manual.md

```diff
@@ -1626,6 +1626,8 @@ the following methods:
   the positional argument, you can specify external dependencies to
   use with `dependencies` keyword argument.
 
+- `cmd_array()` returns the command arguments for the current compiler.
```

Comment:
Probably best to be explicit that the return value is an array object, so perhaps something like "returns an array containing ...".

Is the contents is just how to invoke the compiler?  Or that plus some arguments?

Reply:
> Is the contents is just how to invoke the compiler? Or that plus some arguments?

It's an array containing the compiler command plus any arguments gathered from the `CC` envvar or specified in the cross file. For example:

For example, setting `CC="ccache gcc -m32"` in the environment would cause `cmd_array()` to return `['ccache', 'gcc', -m32']`.

Or, in a cross file:
```
[binaries]
c = 'aarch64-unknown-linux-gnu-gcc'
```
Would produce this result: `['aarch64-unknown-linux-gnu-gcc']`